python/grep.py

```diff
@@ -66,6 +69,11 @@
 #
 #   History:
 #
+#   2017-09-20, mickael9
+#   version 0.7.9:
+#   * use weechat 1.5+ api for background processing (old method was unsafe and buggy)
```

Comment:
Does this change the minimum weechat script version possible? Version bump should at least be minor (or perhaps even major, due to bc break) - definitely not patch.

Reply:
I bumped it to 0.8, but I think users should be discouraged from using weechat <1.9.1 with all the security vulnerabilities